integrations/gem/actions/update-application.ts

```
@@ -0,0 +1,18 @@
+import type { NangoAction, ProxyConfiguration } from '../../models';
```

ESLint

[error] '../../models' imported multiple times.

(import/no-duplicates)